src/test/java/com/github/checkstyle/regression/git/DiffParserTest.java

```diff
@@ -81,6 +98,50 @@ public void testParseDeleteChange() throws Exception {
     }
 
     @Test
+    public void testParseRenameChange() throws Exception {
+        try (Repository repository = GitUtils.createNewRepository()) {
+            GitUtils.addAnEmptyFileAndCommit(repository, "HelloWorld");
+            GitUtils.createNewBranchAndCheckout(repository, "foo");
+            GitUtils.addAnEmptyFileAndCommit(repository, "HelloWorldFoo");
+            GitUtils.removeFileAndCommit(repository, "HelloWorld");
+            final List<GitChange> changes = DiffParser.parse(
+                    repository.getDirectory().getParent(), "foo");
+            assertEquals("There should be 1 change detected", 1, changes.size());
+            final GitChange expected = ImmutableGitChange.builder()
+                    .path("HelloWorldFoo")
+                    .build();
+            assertEquals("The change is not as expected", expected, changes.get(0));
+        }
+    }
+
+    @Test
+    public void testParseCopyChange() throws Exception {
+        try (Repository repository = GitUtils.createNewRepository()) {
+            GitUtils.addAnEmptyFileAndCommit(repository, "a.txt");
+            GitUtils.createNewBranchAndCheckout(repository, "foo");
+            GitUtils.addAnEmptyFileAndCommit(repository, "b.txt");
+            GitUtils.addAnEmptyFileAndCommit(repository, "src/com/foo/c.java");
+            GitUtils.addAnEmptyFileAndCommit(repository, "src/com/foo/d.java");
+            GitUtils.removeFileAndCommit(repository, "a.txt");
```

Comment:
Yep, I removed the code for detecting `RENAME` and `COPY`. Now all of them would be detected as `ADD`.